app/upgrades/v20/upgrades.go

```diff
@@ -0,0 +1,57 @@
+package v20
+
+import (
+	errorsmod "cosmossdk.io/errors"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	stakeibckeeper "github.com/Stride-Labs/stride/v19/x/stakeibc/keeper"
+	"github.com/Stride-Labs/stride/v19/x/stakeibc/types"
+)
+
+const (
+	UpgradeName           = "v20"
+	dydxCPTreasuryAddress = "dydx15ztc7xy42tn2ukkc0qjthkucw9ac63pgp70urn"
+	dydxChainId           = "dydx-mainnet-1"
+)
+
+// CreateUpgradeHandler creates an SDK upgrade handler for v20
+func CreateUpgradeHandler(
+	mm *module.Manager,
+	configurator module.Configurator,
+	cdc codec.Codec,
+	stakeIbcKeeper stakeibckeeper.Keeper,
+) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		ctx.Logger().Info("Starting upgrade v20...")
+
+		ctx.Logger().Info("Adding DYDX Community Pool Treasury Address...")
+		if err := SetDydxCommunityPoolTreasuryAddress(ctx, stakeIbcKeeper); err != nil {
+			return nil, errorsmod.Wrapf(err, "unable to set dydx community pool treasury address")
+		}
```

Comment:
per the slack convo, can you scratch this from this PR so that this PR is just the upgrade handler shell?

Reply:
done